<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetOptionsBatiments</class>
 <widget class="QFrame" name="WidgetOptionsBatiments">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_decription">
     <property name="styleSheet">
      <string notr="true">font-size: 14px;</string>
     </property>
     <property name="text">
      <string>Options du batiments :</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_vendre">
     <property name="text">
      <string> Vendre</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_deplacer">
     <property name="text">
      <string> Deplacer</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_pivoter">
     <property name="text">
      <string> Pivoter</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_retour">
     <property name="text">
      <string> Retour</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_pivoter">
       <property name="text">
        <string>Pivoter :</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_gauche">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_droite">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_confirmer">
       <property name="text">
        <string>Confirmer</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
